brain-marks/Settings/Views/WhatsNewView.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  WhatsNewView.swift
+//  brain-marks
+//
+//  Created by Susannah Skyer Gupta on 10/26/22.
+//
+
+import SwiftUI
+
+struct WhatsNewView: View {
+  @StateObject var viewModel: SettingsViewModel
+
+    var body: some View {
+      let plainString = viewModel.getStringFromBundle()
+      let attributedString = viewModel.createAttributedString(plainString: plainString)
+      Text(attributedString)
+        .padding()
+      Spacer()
+    }
```

Comment:
Can you put this into a `ScrollView`? That will help accommodate as the file gets bigger and with dynamic type. Here's an interesting [article](https://lickability.com/blog/every-screen-in-your-app-should-be-a-scrolling-view/)!

A better way to this would be to have a function called `createWhatsNewAttributedString` that uses the function `getStringFromBundle` rather than needing 2 variables in the `View`

This also makes it so you can make `getStringFromBundle` a private function

Lastly instead of returning a blank string, you could return an error message saying like "Problem with file" idk haha something useful, otherwise an empty string is weird